src/test/java/com/example/calculator/CalculatorApplicationTests.java

```diff
@@ -0,0 +1,69 @@
+package com.example.calculator;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.stream.Stream;
+
+@SpringBootTest
+class CalculatorApplicationTests {
+	private final LogikCalculator calculator = new LogikCalculator();
+
+	@ParameterizedTest
+	@MethodSource("paramsForTestSumma")
+	void summaTest() {
+		int result = calculator.getSum(2,4);
+		Assertions.assertNotNull(result);
+		Assertions.assertDoesNotThrow(() -> result );
+	}
```

Comment:
Данные для теста должны браться из источника.
Также нужно проверять результат работы метода.
Вот пример:

```suggestion
	@ParameterizedTest
	@MethodSource("paramsForTestSumma")
	void summaTest(int a, int b) {
		int result = calculator.getSum(a, b);
                int expected = a + b;
		Assertions.assertEquals(expected, result);
	}
```